frontend/public/locales/en/list-page.json

```diff
@@ -0,0 +1,4 @@
+{
+  "Create {{label}}": "Create {{label}}",
+  "by name": "by name"
+}
```

Comment:
@rawagner Please add newline at the end of file; relates to `insert_final_newline` in `.editorconfig` file.

Reply:
@vojtechszocs I believe i18next-parser does not add a final newline, so it's better to leave it off. It will get added back by the utility next time a message is added anyway.

Having said that, we should generally put these strings in the `public` namespace if they are in the `public` folder.

Reply:
The file is now removed and keys are in `public` ns

Reply:
Thanks @spadgett - I forgot that localization files are managed by i18next parser content-wise.